lib/cloud/azure/vm.go

```diff
@@ -314,7 +315,10 @@ func NewRunCommandClient(subscription string, cred azcore.TokenCredential, optio
 
 // Run runs a command on a virtual machine.
 func (c *runCommandClient) Run(ctx context.Context, req RunCommandRequest) error {
-	poller, err := c.api.BeginCreateOrUpdate(ctx, req.ResourceGroup, req.VMName, "RunShellScript", armcompute.VirtualMachineRunCommand{
+	// TODO(Tener): make the run command name actual parameter.
+	const runCommandName = "teleport-install"
```

Comment:
What's the purpose of this name change?

Reply:
The command we create lives in a global scope; we should give it reasonable name, associated with Teleport. Given that we only use this interface for installing Teleport, this name makes sense. In ideal world this should depend on some upstream parameters, like install suffix or the agent performing the installation. There are some additional bits of work that need to be done before we can do it: removing old run commands, making sure we are racing the installation from two discovery agents etc.